.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+install:
+    # Requirements needed for Travis CI
+    - pip install coveralls
+    - pip install yapf
```

Comment:
We need explicit versions for `yapf` and `pyflakes` so that we don't break if they add new functionality. Can we explicit versions (using the latest available) for each? Also, plyint is on `1.7.2` now.

Coveralls is fine using latest and greatest because it's hard for that to break us on update.